apps/frontend/src/components/TimeCardPage/TimeSheet.tsx

```diff
@@ -350,6 +353,31 @@ export default function Page() {
 
   // use this to control whether the timesheet is disabled or not
   const disabled = false;
+  const [isOpenCommentForm, setIsOpenCommentForm] = useState(false);
+  const [comments, setComments] = useState<Comment[]>([]);
```

Comment:
Is there a reason for having the list of comments stored and set here, rather than doing something like the timesheets where we simply save the timesheet with the updated comments and then pass the `currentTimesheet`'s weekly comments into the `Evaluations` component?

Reply:
I think this might be okay to leave for now, just with a TODO comment to replace it with the correct call to our API, but that's a separate ticket for next sprint.